tests/components/husqvarna_automower/conftest.py

```diff
@@ -0,0 +1,86 @@
+"""Test helpers for Husqvarna Automower."""
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from aioautomower.model import MowerAttributes, MowerList
+from aioautomower.session import AutomowerSession
+import pytest
+
+from homeassistant.components.husqvarna_automower.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from .common import setup_integration
+
+from tests.common import MockConfigEntry, load_fixture, load_json_value_fixture
+
+TEST_MOWER_ID = "c7233734-b219-4287-a173-08e3643f89f0"
+
+
+@pytest.fixture(scope="session")
+def load_mower_fixture():
+    """Load Fixture data."""
+    return load_json_value_fixture("mower.json", DOMAIN)
+
+
+@pytest.fixture(scope="session")
```

Comment:
I suggest we don't scope this for the session so that we can modify this data freely in each test without worrying about affecting other tests.

```suggestion
@pytest.fixture
```